src/drivers/clock/e2k/e2k.c

```diff
@@ -71,13 +72,14 @@
 
 static struct clock_source e2k_clock_source;
 static irq_return_t e2k_clock_handler(unsigned int irq_nr, void *dev_id) {
+	static int k = 0;
```

Comment:
Unused variable?

Reply:
Thanks, I removed this and the next commented out line.